src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchCaseSearchOperation.java

```diff
@@ -43,22 +45,25 @@
     private final CaseDetailsMapper caseDetailsMapper;
     private final ApplicationParams applicationParams;
     private final CaseSearchRequestSecurity caseSearchRequestSecurity;
+    private final MergeDataToSearchCasesOperation mergeDataToSearchCasesOperation;
 
     @Autowired
     public ElasticsearchCaseSearchOperation(JestClient jestClient,
                                             @Qualifier("DefaultObjectMapper") ObjectMapper objectMapper,
                                             CaseDetailsMapper caseDetailsMapper,
                                             ApplicationParams applicationParams,
-                                            CaseSearchRequestSecurity caseSearchRequestSecurity) {
+                                            CaseSearchRequestSecurity caseSearchRequestSecurity,
+                                            MergeDataToSearchCasesOperation mergeDataToSearchCasesOperation) {
         this.jestClient = jestClient;
         this.objectMapper = objectMapper;
         this.caseDetailsMapper = caseDetailsMapper;
         this.applicationParams = applicationParams;
         this.caseSearchRequestSecurity = caseSearchRequestSecurity;
+        this.mergeDataToSearchCasesOperation = mergeDataToSearchCasesOperation;
     }
 
     @Override
-    public CaseSearchResult execute(CrossCaseTypeSearchRequest request) {
+    public CaseSearchResult executeExternal(CrossCaseTypeSearchRequest request) {
```

Comment:
I would keep the code method and classes agnostic to the external vs internal classification